docs/src/main/paradox/release-notes/10.2.x.md

```diff
@@ -1,5 +1,41 @@
 # 10.2.x Release Notes
 
+## 10.2.8
+
+This release is a maintenance release, mostly fixing a few smaller issues around HTTP/2 support.
+
+Akka HTTP now also supports graceful termination for HTTP/2 servers. If coordinated shutdown is enabled for a server binding
+via `ServerBinding.addToCoordinatedShutdown`, the server will allow ongoing requests to be completed for a configurable
+period of time while immediately rejecting incoming requests before shutting down the application process.
+This helps to avoid downtime during rolling upgrades.
+
+#### akka-http-core
+
+* Enable idle-timeout when http2 is enabled [#3965](https://github.com/akka/akka-http/pull/3965)
+* Fix h2c via Upgrade [#4000](https://github.com/akka/akka-http/pull/4000)
+* Fail pool slot if connection is canceled before it is failed [#3981](https://github.com/akka/akka-http/pull/3981)
+* Ensure PoolMaster.shutdown(…) does not hang [#4004](https://github.com/akka/akka-http/pull/4004)
+* Implement graceful termination for HTTP/2 servers [#4018](https://github.com/akka/akka-http/pull/4018)
+* Remove unused var from ServerTerminator [#4011](https://github.com/akka/akka-http/pull/4011)
+
+#### akka-http
+
+* Interpret SSE heartbeats as events [#4020](https://github.com/akka/akka-http/pull/4020)
+
+#### docs
+
+* Simplify bullet points in 10.2.7 release notes[#3926](https://github.com/akka/akka-http/pull/3926)
```

Comment:
```suggestion
* Simplify bullet points in 10.2.7 release notes [#3926](https://github.com/akka/akka-http/pull/3926)
```

Reply:
Oops, too fast with the trigger. https://github.com/akka/akka-http/pull/4035